doc/docs/modules/ROOT/pages/neo4j-cluster.adoc

```diff
@@ -2,24 +2,21 @@
 [#neo4j_causal_cluster]
 = Using with Neo4j Causal Cluster
 
-[abstract]
---
-This chapter describes considerations around using Neo4j Connector for Apache Spark with Neo4j Enterprise Causal Cluster.
---
+:description: This chapter describes considerations around using Neo4j Connector for Apache Spark with Neo4j Enterprise Causal Cluster.
 
 == Overview
 
 link:https://neo4j.com/docs/operations-manual/current/clustering/[Neo4j Clustering] is a feature available in
-Enterprise Edition which allows high availability of the database through having multiple database members.
+Enterprise Edition, which allows high availability of the database through having multiple database members.
 
-Neo4j Enterprise uses a link:https://neo4j.com/docs/operations-manual/current/clustering/introduction/#causal-clustering-introduction-operational[LEADER/FOLLOWER]
-operational view, where writes are always processed by the leader, while reads can be serviced by either followers,
-or optionally be read replicas, which maintain a copy of the database and serve to scale out read operations
+Neo4j Enterprise uses a link:https://neo4j.com/docs/operations-manual/current/clustering/introduction/#causal-clustering-introduction-operational[Primary/Secondary servers]
```

Comment:
@conker84, please, pay attention to this paragraph. I've consulted with the clustering team, they said it's not wrong, but confusing, because terms of Primary/Secondary servers clash with terms of Leader/Followers and Read Replicas. So I tried to fix it. Maybe you can suggest other options for editing